be/src/runtime/exec_env.cpp

```diff
@@ -487,13 +488,15 @@ Status ExecEnv::init(const std::vector<StorePath>& store_paths, bool as_cn) {
         });
         config::starlet_cache_dir = JoinStrings(starlet_cache_paths, ":");
     }
+    _lake_pk_index_loader = new lake::PkIndexLoader();
```

Comment:
need to take care of the shutdown process, when need to delete this PkIndexLoader?

Reply:
done